spec/features/authorization_spec.rb

```diff
@@ -1,17 +1,12 @@
 require 'rails_helper'
 
-      #save_and_open_page
```

Comment:
In case you were wondering, this are littered about so I can copy and paste when I needed them. There should go now, though.